src/pykiso/lib/auxiliaries/udsaux/uds_auxiliary.py

```diff
@@ -193,12 +195,13 @@ def send_uds_config(
             req_resp_data = uds_service(**msg_to_send["data"])
             if req_resp_data is None:
                 req_resp_data = True
-            log.internal_info(f"UDS response received {req_resp_data}")
+            log.internal_info("UDS response received %s", req_resp_data)
             return req_resp_data
         except AttributeError:
             # Service not found, raised by getattr()
             log.exception(
-                f"Could not send UDS config request: unknown service {uds_service_name}"
+                "Could not send UDS config request: unknown service %s",
```

Comment:
It makes less sense to have percent formatting for warning and error levels as these are usually never filtered out

Reply:
It has been reverted